go/state/cppstate/cpp_state.go

```diff
@@ -139,7 +139,9 @@ func (cs *CppState) SetStorage(address common.Address, key common.Key, value com
 	update.AppendSlotUpdate(address, key, value)
 	return cs.Apply(0, update)
 }
-
+func (cs *CppState) HasEmptyStorage(common.Address) (bool, error) {
+	panic("HasEmptyStorage: not implemented for CppState")
```

Comment:
This will let all Aida tests targeting a C++ implementation fail because they will still ask for the storage state.

For replying Fantom history, returning `true` here should be safe.

A cleaner alternative would be to define a "ErrEmptyStateNotImplemented" in the file defining the `State` interface and allow this error to be returned. At the calling this error should be checked and ignored by returning the "safe" fallback of `true` -- which should be valid for all Fantom networks.

However, a TODO + issue should be added requesting the implementation of this function.